lib/wanda/catalog/resolved_value.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Wanda.Catalog.ResolvedValue do
+  @moduledoc """
+  Represents a resolved check value as defined check specification.
+  It is based on the contextual environment.
+  """
+  alias Wanda.Catalog.Value
+
+  @type value :: boolean() | number() | String.t()
+
+  @derive Jason.Encoder
+  defstruct [
+    :spec,
+    :name,
+    :original_value,
```

Comment:
Remember to change this if you decide to change the other in the `SelectableCheck`